### ISSUE TITLE
[Praise+small issue] Works really well when compiled!

### ISSUE THREAD

**Issue by Merith-TK** [termux/termux-x11#13]
I compiled this for [Another guy](https://github.com/termux/termux-api/issues/257#issuecomment-630169022)

and decided to try it out, it works like a charm aside from two issues that really arnt too big of a deal

1. `Xwayland` just crashes occasionally when using xfce4
I have not tested with other DE's because my phone doesn't have alot of space, but with xfce4, it will suddenly crash when loading a gtk3 based application. It doesnt do it all of the time, it just kinda happens at random, and only when a gtk3 app is loading
```
Xwayland :1 
<open second session>
DISPLAY=:1 xfce4-session
```
![](https://cdn.discordapp.com/attachments/449115653958270976/712667889794023504/Screenshot_20200520-070738_Termux.jpg)

2. No good keyboard input without external keyboard
Self explanatory, the `florence` keyboard on the repo does the job, but it isnt a good keyboard to use for this case.

**Comment by Merith-TK** [comment]
and yes i know it is *unreleased*

doesnt mean people cant use it and report issues they find.

**Comment by kcubeterm** [comment]
Can you provide your device info, i don't encounter any issues with xfce ,

**Comment by Merith-TK** [comment]
sure, give me a short bit
probably has to do with the fact that my phone has an armv8l cpu, but OEM samsung was a dick and put ARM Android on there instead of ARM64 android

**Comment by kcubeterm** [comment]
O. yes, @xeffyr also mentioned crashes in arm devices

**Comment by Merith-TK** [comment]
yep. figured. Eh it works quite well. and the random crashes dont happen too often.

the armv7 device crashes alot more than my phone, but it works well enough to do what i need it to do (had it running for several hours and didnt crash)
